### ISSUE TITLE
Verse and Chorus (only) section labels incorrectly omitted if lower case (UG parser only)

### ISSUE THREAD

**Issue by eplebel** [martijnversluis/ChordSheetJS#942]
Verse and Chorus (only) section labels are incorrectly omitted if they're lower case.

That is, non-verse & non-chorus section names correctly appear (whether in lowercase or upper case) whereas verse and chorus section names only appear in upper case. Examples:

Correctly appear:
"[Intro]" or "[INTRO]"
"[Bridge]" or "[BRIDGE]"
"[VERSE 1]"
"[CHORUS]"

Missing/do not appear:
"[Verse 1]"
"[Chorus]"

The issue does NOT occur with the Chord pro parser (const parser = new ChordSheetJS.ChordProParser(); )

This was mentioned as a comment to a possibly related issue #922 (UG parser using deprecated ChordSheetParser), but logging it as a separate issue given it's possible it's an UNRELATED issue.

**Comment by spiffylogic** [comment]
Looks like this is because of the case-sensitive regex pattern found in `ultimate_guitar_parser.ts`:
```
const VERSE_LINE_REGEX = /^\[Verse.*]/;
const CHORUS_LINE_REGEX = /^\[Chorus]/;
```
So `Verse` or `Chorus` gets absorbed as the `type` of the lines that follow. Any other label (like Intro or VERSE) gets parsed and rendered as a comment. @martijnversluis can you confirm if this is a bug?

**Comment by martijnversluis** [comment]
@eplebel, @spiffylogic,

It might be a bug. When I check the [UG tab docs](https://www.ultimate-guitar.com/contribution/help/rubric#ii4) all examples of tags start with an uppercase letter. However, requirements on the casing are nowhere to be found.

It probably makes sense to make the regexes case-insensitive. I'll add it to the [project board](https://github.com/martijnversluis/ChordSheetJS/projects/1).

**Comment by spiffylogic** [comment]
> It probably makes sense to make the regexes case-insensitive. I'll add it to the [project board](https://github.com/martijnversluis/ChordSheetJS/projects/1).

@martijnversluis That looks like a simple change (I drafted a PR #961) but then _all_ verse and chorus tags are not rendered, which is the opposite of what @eplebel was looking for. I guess the question here is why are these tags not rendered by the formatter?

**Comment by martijnversluis** [comment]
Resolved in #961

**Comment by martijnversluis** [comment]
@eplebel, @spiffylogic I just published `v8.0.0` that contains @spiffylogic's bugfix. Please let me know if that solves your issue 👍

**Comment by martijnversluis** [comment]
@spiffylogic Sorry, I totally misread the issue. I reopened the issue, I'll try to take a look at this shortly 👍

**Comment by martijnversluis** [comment]
Btw @spiffylogic Which formatter do you use?

**Comment by spiffylogic** [comment]
> Btw @spiffylogic Which formatter do you use?

@martijnversluis `HtmlDivFormatter`

**Comment by martijnversluis** [comment]
@eplebel , @spiffylogic I published `v8.5.0` with a fix for this.
